 NOTICE.txt

```diff
@@ -0,0 +1,49 @@
+﻿MarkLogic® for Node.js Client API
+
+Copyright © 2022-2023 MarkLogic Corporation.  MarkLogic and MarkLogic logo are trademarks or registered trademarks of MarkLogic Corporation in the United States and other countries.  All other trademarks are the property of their respective owners.
+
+This project is licensed under the Apache License, Version 2.0 (the "License"); you may not use this project except in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+To the extent required by the applicable open-source license, a complete machine-readable copy of the source code corresponding to such code is available upon request. This offer is valid to anyone in receipt of this information and shall expire three years following the date of the final distribution of this product version by MarkLogic Corporation. To obtain such source code, send an email to Legal-thirdpartyreview@progress.com. Please specify the product and version for which you are requesting source code.
+
+The following software may be included in this project (last updated October 3, 2023):
+
+-------------------------------------------------------------------------
+Third Party Components

```

Comment:
This doesn't look correct - these look like the dependencies for the documentation generator under `etc/marklogic-template`. Shouldn't this have the third party dependencies listed in `./package.json`?

Reply:
This was given by the legal team.